test/contracts-network/colony-expenditure.js

```diff
@@ -380,20 +369,47 @@ contract("Colony Expenditure", (accounts) => {
       );
     });
 
-    it.skip("should not allow owners to set a non-global/local skill or a deprecated global skill", async () => {
```

Comment:
I'm honestly not 100% sure what this test was intending to test. So I've added two more that cover what I think might have been our intention.

Reply:
These tests are duplicates of existing tests, see line https://github.com/JoinColony/colonyNetwork/blob/develop/test/contracts-network/colony-expenditure.js#L277